src/style.css

```diff
@@ -1,3 +1,107 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  margin-left: 20px;
+}
+
+/* Use Roboto font */
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+/* Header styling */
+header {
+  background-color: #fff;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+/* Logo styling */
+.logo img {
```

Comment:
do not use combined and tag selectors
Just add class to img tag and use it to add styels